grid/webhook-management/subscribe-webhook/map/shopify.suma.js

```diff
@@ -0,0 +1,49 @@
+function WebhookSubscription({ input, parameters, services }) {
+  const createWebhookResult = createWebhook(input, services, parameters);
+
+  return createWebhookResult;
+}
+
+function createWebhook(input, services, parameters) {
+  const url = `${services.default}admin/api/${parameters.api_version}/webhooks.json`;
+  const options = {
+    method: 'POST',
+    body: input,
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    security: 'apiKey',
+  };
+
+  const response = std.unstable.fetch(url, options).response();
+  const body = response.bodyAuto() ?? {};
+
+  if (response.status !== 201) {
+    throw new std.unstable.MapError({
+      errors: [
+        {
+          code: response.status.toString(),
+          message: 'Failed to create webhook',
+          options: body,
```

Comment:
nitpick: I think this is wrong - body is not necessarily an object as profile requires. Also I think the name of the field is unclear.

Reply:
Yes, I missed this.